src/graphql/data/nft/NftBalance.ts

```diff
@@ -136,10 +137,26 @@ export function useNftBalanceQuery(
   )
   const walletAssets: WalletAsset[] = data.nftBalances?.edges?.map((queryAsset: NftBalanceQueryAsset) => {
     const asset = queryAsset.node.ownedAsset
+    const ethPrice = parseEther(
+      asset?.listings?.edges[0]?.node.price.value?.toLocaleString('fullwide', { useGrouping: false }) ?? '0'
+    ).toString()
     return {
       id: asset?.id,
-      image_url: asset?.image?.url,
-      image_preview_url: asset?.smallImage?.url,
+      imageUrl: asset?.image?.url,
+      smallImageUrl: asset?.smallImage?.url,
+      address1: asset?.collection?.nftContracts?.[0].address,
```

Comment:
instead of `address1` you already have this information in `asset_contract.address`

Reply:
done